src/scrabble.js

```diff
@@ -1,5 +1,57 @@
-function scrabble() {
+// Each letter has a numerical value. Write code that takes a word and return the total numerical sum of each letter.
 
+// Psudocode
+// 1.) Create an array with all letters
+// 2.) Create function with loads of if statements to get numerical value. e.g if B (index 1 in above array) return 3. 
+// 3.) Store this data somewhere (object maybe?)
+// 4.) Create a function that takes a string as an argument
+// 5.) For loop within function that loops round the length of the string
+// 6.) Within the for loop translate each index of the string to it's numerical value (how??? - Maybe function from point 2 (index[i]) *Need to think of scope here*
+// 7.) Store these values in array
+// 8.) Loop around array to add
+// 9.) Return value 
+
+const lettersArray = ['A','B','C','D','E','F','G','H','I','J','K','L','M','N','O','P','Q','R','S','T','U','V','W','X','Y','Z']
```

Comment:
This part seems like it ended up not being needed? I did a similar thing though so...

Reply:
The array is indeed useless. That being said, another data structure could be used instead